data/data-pipeline/data_pipeline/content/config/csv.yml

```diff
@@ -14,6 +14,30 @@ fields:
   - score_name: State/Territory
     label: State/Territory
     format: string
+  - score_name: Percent Black or African Amer.
```

Comment:
nit: can we spell this out?

Reply:
I'm just following @KameronKerger 's field names!